pkg/apis/flowcontrol/types.go

```diff
@@ -419,6 +419,10 @@ type LimitedPriorityLevelConfiguration struct {
 	// Bigger numbers mean a larger nominal concurrency limit,
 	// at the expense of every other priority level.
 	// This field has a default value of 30.
+	//
+	// Setting this field to zero allows for the construction of a
```

Comment:
can we say "supports" rather than "allows"?  I think "allows" suggests that this is _all_ that is needed to make that jail, and this is not true.